views/error.hbs

```diff
@@ -8,32 +8,37 @@
     <meta property="og:image" content="http://www.ideahacks.la/img/meta_thumbnail.png">
     <meta property="og:description" content="For the seventh time, UCLA IEEE and the UCLA Chapter of Theta Tau are pleased to present IDEA Hacks, the largest hardware focused hackathon on the west coast. Apply on our website today!">
 
+
+    <!-- BOOTSTRAP -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+
     <!-- GOOGLE FONTS -->
-    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;700&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&family=Source+Code+Pro:wght@400;500;700&display=swap" rel="stylesheet" />
 
     <!-- PERSONAL STYLESHEETS -->
     <link rel="stylesheet" type="text/css" href="/css/style.css" />
     <link rel='stylesheet' type='text/css' href='/css/error-page.css' />
 
-    <!-- BOOTSTRAP -->
-    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
 </head>
 
 
-<body style="background-color: rgba(209, 235, 250, 0.7);">
+<body>
+    <div class="wrapper">
+        <!-- NAVBAR -->
+        {{> navbar }}
```

Comment:
The reason the navbar doesn't have anything in it is because the Handlebars for the include doesn't check for the 404 page. This is on me. I forgot that we don't use it on the 404 page. You can just remove the navbar. Same goes for the footer.

Reply:
Should I leave in the footer? I added it in along with the header in my previous commit.

Reply:
Nope, you can remove it too